src/database/migrations/20190914173000-modify-request-table-with-new-columns.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => Promise.all([
+    queryInterface.addColumn(
+      'Requests',
+      'passportNumber', {
+        type: Sequelize.STRING,
+        allowNull: true
+      }
+    ),
+    queryInterface.addColumn(
+      'Requests',
+      'passportName', {
+        type: Sequelize.STRING,
+        allowNull: true
+      }
+    ),
+    queryInterface.addColumn(
+      'Requests',
+      'lineManager', {
+        type: Sequelize.STRING,
+        allowNull: true
+      }
+    ),
+    queryInterface.addColumn(
```

Comment:
There's an association with the request and booking table already, I'm not sure if we need this field on request table